doc/Release_Build_Creation.rst

```diff
@@ -4,6 +4,23 @@ Release Build Creation
 
 This file consists of the procedure that should be followed to generate the release build outside the Seagate network using `cortx-build <https://github.com/orgs/Seagate/packages/container/package/cortx-build>`_ docker image. 
 
+***************
+### Hardware Specifications - Single-Node VM deployment
```

Comment:
@christina-00 I feel these are specifications for Deployment and not for compilation. So we should keep them separate. 
e.g. There may be case where developer can use dedicated system for compilation and different nodes for deployment.

Reply:
Actually i follow the guide from **Release Build Creation** then go to **Provision Release Build**. i had to redo everything when i did not fulfill the requirement (3 network interface and 2 disks) in **Provision Release Build**. I am hoping to avoid this mistake for others going through the guideline. Do you think we can slot in this information somewhere else?

Reply:
Ok. May be we can add this info in Provision Release Build.  Then link them here and mention that if you are planning to compile and deploy on same system it should satisfy mentioned requirements.

Reply:
I hope lets update in both the Wiki's i.e. Build Creation & Release Build to avoid any confusion for community

Reply:
I feel we should avoid duplication on documentation and reuse existing ones wherever possible.